pkg/plugin/provider/certs.go

```diff
@@ -212,32 +212,58 @@ func RemoveCerts(namespace string) error {
 func UnImportCertificate(file certFile, namespace string) error {
 	dynamicDir := fmt.Sprintf("%s/%s", gmsaDirectory, namespace)
 
-	// get cert thumbprint using certutil. Thumbprints are equal to the sha1 hash of the certificate
+	logrus.Infof("Attempting to remove certificate %s", file.hostFile)
+
+	// get cert thumbprint using certutil. Thumbprints are equal to the sha1 hash of the certificate.
+	// Cert util will return a sha1 field in the following format
+	// Cert Hash(sha1): <someHash>
 	certUtilArgs := []string{"-Command",
-		fmt.Sprintf("$(certutil %s)", file.hostFile), "-like", "\"Cert Hash(sha1):*\""}
+		fmt.Sprintf("certutil %s", fmt.Sprintf("C:%s", file.hostFile)), "|", "Select-String", "-Pattern", "\"Cert Hash\\(sha1\\)\"",
```

Comment:
```suggestion
	         "certutil",
	         filepath.Join("C:", file.hostFile),
		"|",
		"Select-String",
		"-Pattern",
		`"Cert Hash\(sha1\)"`,
```

Couple of nits:
1. certutil should be on it's own line in the args (or maybe it shouldn't be in a variable called `certUtilArgs`... Maybe it should be in `certUtilCmd`?)
2. Ideally with multiple lines in a `[]string` object, we should have them be on different lines
3. Use "`" for cleaner escaping

Reply:
Well, this array does represent a list of arguments passed to powershell to invoke `certutil`, which was my initial reasoning for the name. 

Putting the `certutil` command onto its own line sounds good, but I think putting every element onto its own line makes the command harder to read.